Wrangler configuration issue

Running `npx wrangler dev` generates an error:
```
✘ [ERROR] Processing wrangler.toml configuration:

    - r2_buckets[0].preview_bucket_name= "" is invalid. Bucket names must begin and end with an
  alphanumeric character, only contain lowercase letters, numbers, and hyphens, and be between 3 and
  63 characters long.
```

--- a/worker/wrangler.toml
+++ b/worker/wrangler.toml
@@ -5,7 +5,7 @@ compatibility_date = "2024-08-21"
 compatibility_flags = ["nodejs_compat"]
 
 r2_buckets = [
-  { binding = "SESSIONS_BUCKET", bucket_name = "meetecho-sessions", preview_bucket_name = "" }
+  { binding = "SESSIONS_BUCKET", bucket_name = "meetecho-sessions" }
 ]
 
 # Automatically place your workloads in an optimal location to minimize latency.

